java/src/org/openqa/selenium/support/ui/SlowLoadableComponent.java

```diff
@@ -36,11 +36,15 @@ public abstract class SlowLoadableComponent<T extends LoadableComponent<T>>
     extends LoadableComponent<T> {
 
   private final Clock clock;
-  private final Duration timeOutInSeconds;
+  private final Duration timeOut;
 
-  public SlowLoadableComponent(java.time.Clock clock, int timeOutInSeconds) {
+  public SlowLoadableComponent(Clock clock, int timeOutInSeconds) {
+    this(clock, Duration.ofSeconds(timeOutInSeconds));
+  }
```

Comment:
Why don't you deprecate this one so we eventually only leave the new one?

Reply:
And perhaps add a JavaDoc to tell the user.

Reply:
Sure, done!

Reply:
Thank you, @kluever!